lib/themes/dosomething/paraneue_dosomething/js/utilities/Analytics.js

```diff
@@ -42,9 +42,19 @@ function subscribeEvents() {
   });
 }
 
+function exists() {
```

Comment:
@DFurnes just want to be sure. Does this need to be exported if it's used in the below function that definitely is exported? Or will magic ES6 know to grab this function and cram it into the exported function below?